chocolatey/tools/chocolateyUninstall.ps1

```diff
@@ -13,7 +13,6 @@
         foreach($line in $oldProfile) {
             if ($line -like '*PoshGitPrompt*') { continue; }
             if ($line -like '*Load posh-git example profile*') { continue; }
-            if ($line -like '*Start-SshAgent*') { continue; }
```

Comment:
Hrm... I was going to say we should leave this, to clean up after old versions, but this uninstall script will only be used for 1.0+ which will have no responsibility for `Start-SshAgent`...so I think it's fine to go? Thoughts?

Reply:
Yes good idea.